vllm/entrypoints/openai/rpc/client.py

```diff
@@ -180,18 +193,29 @@ def to_proxy_socket(self):
         finally:
             socket.close(linger=0)
 
-    async def _send_get_data_rpc_request(self, request: RPCUtilityRequest,
+    async def _send_get_data_rpc_request(self,
+                                         request: RPCUtilityRequest,
                                          expected_type: Any,
-                                         error_message: str) -> Any:
+                                         error_message: str,
+                                         timeout: Optional[int] = None) -> Any:
```

Comment:
I think we can drop the `timeout` argument if we are using the global right?

Reply:
We should have one or the other IMO

Reply:
Yeah that's true, looks like this arg is unused